change/@microsoft-teams-js-5681fcee-94b0-459c-ac25-ec6377394b5d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add private MessageChannels capability",
```

Comment:
Please use past tense in the comment. `Added`

Reply:
Updated.